test/extended/util/framework.go

```diff
@@ -2066,6 +2066,11 @@ func DoesApiResourceExist(config *rest.Config, apiResourceName string) (bool, er
 		return false, err
 	}
 	for _, apiResourceList := range allResourceList {
+		gvns := strings.Split(groupVersionName, "/")
+		gvs := strings.Split(apiResourceList.GroupVersion, "/")
```

Comment:
Will the same work if you us `apiResource.Group` instead in the loop bellow? E.g.
```go
if apiResource.Name == apiResourceName && apiResource.Group == groupVersionName {
	return true, nil
}
```